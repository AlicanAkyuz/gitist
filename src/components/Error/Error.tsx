import React, { FunctionComponent, Fragment } from 'react';

// type Props = JSX.IntrinsicElements['button'] & {
//   primary?: boolean;
//   secondary?: boolean;
// };

const Error: FunctionComponent = () => (
  <Fragment>
    <header>Header</header>
    {/* <Routes /> */}
    <footer>Footer</footer>
  </Fragment>
);

export default Error;
