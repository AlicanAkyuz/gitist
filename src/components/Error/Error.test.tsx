import React from 'react';
import { shallow } from 'enzyme';

import Error from './Error';

it('Error corrently renders an error message', () => {
  const wrapper = shallow(<Error />);
  expect(wrapper).toMatchSnapshot();
});
