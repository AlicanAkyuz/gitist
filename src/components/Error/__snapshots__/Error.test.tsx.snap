// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error corrently renders an error message 1`] = `
<p>
  Bir sorun olu≈ütu!
</p>
`;
